pkg/datastore/db_user.go

```diff
@@ -134,8 +139,8 @@ func getDbUsers(tableName string, withTenantIdCheck, withInstanceIdCheck bool) [
 		dbUsers[i].password = getPassword(string(dbUsers[i].username))
 		dbUsers[i].policyName = getRlsPolicyName(string(dbUsers[i].username), tableName)
 	}
-	TRACE("Returning DB user specs for table %q:\n\twithTenantIdCheck - %t\n\twithInstanceIdCheck -  %t\n\tdbUsers - %+v\n",
-		tableName, withTenantIdCheck, withInstanceIdCheck, dbUsers)
+	TRACE("Returning DB user specs for table %q:\n\t[roleT=%s, roleI=%s, tableT=%s, tableI=%s]\n\tdbUsers - %+v\n",
+		tableName, tenantRole, instancerRole, isTableTenanted, isTableInstanced, dbUsers)
```

Comment:
I would change this statement to this:
```go
	TRACE("Returning DB user specs for table %q:\n\t[tenantRole=%t, instancerRole=%t, isTableTenanted=%t, isTableInstanced=%t]\n\tdbUsers - %+v\n",
		tableName,
		tenantRole,
		instancerRole,
		isTableTenanted,
		isTableInstanced,
		dbUsers)
```

A developer reading the logs won't understand what `roleT`, `tableT` and others mean without looking at the code. What I'm suggesting is somewhat better.

Reply:
noted will look into later